src/parser/GeoPoint.h

```diff
@@ -70,25 +61,22 @@ class GeoPoint {
   // Convert the value of this GeoPoint object to a single bitstring.
   // The conversion will reduce the precision and thus change the value.
   // However the lost precision should only be in the range of centimeters.
-  T toBitRepresentation();
+  // Guarantees to only use the lower numDataBits (currently 60 bits),
```

Comment:
```suggestion
  // Guarantees to only use the lower `numDataBits` (currently 60 bits),
```
(we use backticks for variable and type names to distinguish them from "ordinary" words)